src/resources/scss/components/_ng-wig.scss

```diff
@@ -171,6 +171,8 @@
     -moz-appearance: none;
 
     appearance: none;
+    
+    overflow: none;
```

Comment:
There is no `none` value for the `overflow` property.
I guess you want to hide the overflow? Then it would be `hidden`. If you want to reset the property to its original state it would be `initial`. See [w3schools](https://www.w3schools.com/cssref/pr_pos_overflow.asp) for further reference.

Reply:
My bad, I was tired yesterday. I've changed it